File: 1/1.php
<?php

$sumnum = 0;
$num    = 1000;
for( $i=0; $i<$num; $i++ )
{
  if( $i % 3 == 0 || $i % 5 == 0 )
    $sumnum += $i;
}

echo $sumnum."\n";

?>

File: x/1/1.cpp
#include <iostream>

using namespace std;

int main( )
{
  int sumnum = 0;
  int num    = 1000;

  for( int i=0; i<num; i++ )
  {
    if( i % 3 == 0 || i % 5 == 0 )
      sumnum += i;
  }

  cout << sumnum << "\n";
  return 0;
}
